### ISSUE TITLE
Fatal error on SSL transport

### ISSUE THREAD

**Issue by ewtoombs** [aaugustin/websockets#356]
I'm not exactly sure what caused this error, but I was a client receiving messages on a websocket for a while (about 12 hours). Suddenly all incoming data stopped, then nothing happened for about 5 hours. Finally, I received a ConnectionClosed and the following appeared on stdout:
```
Fatal error on SSL transport
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f17ae393e48>
transport: <_SelectorSocketTransport closing fd=15 read=idle write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/sslproto.py", line 636, in _process_write_backlog
    ssldata, offset = self._sslpipe.feed_appdata(data, offset)
AttributeError: 'NoneType' object has no attribute 'feed_appdata'
```
I can't imagine this is what was supposed to happen. I'll let you know if it happens again. I have no idea how to reproduce this reliably.

**Comment by aaugustin** [comment]
It's very unclear which layer is responsible for this issue; it's probably a lower-level layer than websockets, though.

**Comment by ewtoombs** [comment]
It has happened again. I agree it's really hard to figure out the cause. Any ideas as to how to debug this further? Meanwhile, I'm sending this upstream to see if they know what's causing it.

**Comment by ewtoombs** [comment]
I wouldn't mind running a branch of websockets made to debug this problem, if you made one.

**Comment by Harmon758** [comment]
This was an [issue with asyncio](https://bugs.python.org/issue33037) and should be fixed in Python 3.6.5 now.
See also: https://bugs.python.org/issue33093

**Comment by ewtoombs** [comment]
Whoops, I forgot to tell you guys. I actually knew that, lol.
